irohad/ametsuchi/impl/peer_query_wsv.cpp

```diff
@@ -15,9 +15,10 @@
  * limitations under the License.
  */
 
-#include <utility>
-
 #include "ametsuchi/impl/peer_query_wsv.hpp"
+#include <utility>
```

Comment:
I'd prefer to order includes like Google (https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes).